pages/algorithms/semantic-role.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Semantic Role 
+key: semantic-role
+---
+
+A semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
```

Comment:
editorial: replace with "A semantic role is a semantic association that indicates the type of an object. It allows ..."